blueprints/ember-i18next/index.js

```diff
@@ -4,6 +4,9 @@ module.exports = {
   normalizeEntityName: function () {},
 
   afterInstall: function(options) {
-    return this.addBowerPackageToProject('i18next', '^1.7.0');
```

Comment:
Please align the closing braces (`]);`) with the `return` and only indent the array contents one level.